debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/J9BCUtil.java

```diff
@@ -610,6 +610,18 @@ public static void j9bcutil_dumpRomClass(PrintStream out, J9ROMClassPointer romC
 			}
 		}
 
+		/* Permitted subclasses for a sealed class */
+		if (J9ROMClassHelper.isSealed(romClass)) {
+			int permittedSubclassCount = OptInfo.getPermittedSubclassCount(romClass);
+			out.append(String.format("Permitted subclasses (%d):" + nl, permittedSubclassCount));
```

Comment:
Please instead use `out.format("Permitted subclasses (%d):%n", permittedSubclassCount)`
* `%n` expresses the desired newline
* there's no need to build the string locally